### ISSUE TITLE
Remove watchtower in runtime

### ISSUE THREAD

**Issue by fiksn** [lightningnetwork/lnd#6245]
### Background

I've tried using an altruistic watchtower https://lightningnetwork.plus/watchtower
Occasionally I can see a batch of similar error messages in the logs:
```
2022-02-08 23:42:15.021 [ERR] WTCL: (anchor) SessionQueue(03e93bd1d5befc163acaded21568fd77650500d9b5b88c4dccf4607e070cf293b0) unable to dial tower at 0301135932e89600b3582513c648d46213dc425c7666e3380faa7dbb51f7e6a3d6@44.237.235.163:9911: read tcp 192.168.1.159:47676->44.237.235.163:9911: i/o timeout
```
networking else seems to be unaffected from PoV of my node. Is my assumption correct that this is likely related to that watchtower node (possibly being overloaded or something)? I believe that during those short periods also HTLCs my node is supposed to forward timeout. Can this be related to the problems with the watchtower connectivity or is this some spurious correlation?

Regarding LND my issue now is how to remove this watchtower.

I've tried with:
`lncli wtclient remove 0301135932e89600b3582513c648d46213dc425c7666e3380faa7dbb51f7e6a3d6` but always get back:
`[lncli] rpc error: code = Unknown desc = tower has unacked updates`

Is there any way to forcefully remove a watchtower without restarting LND with `wtclient.active=false`.
(I am not even sure if that is safe.) What I want to achieve is simply send a (signed) message to the watchtower "don't bother anymore with sending penalty transactions for me". Or at least how can I get more details about what updates are unacked, since it seems after I enabled this watchtower for test purposes there is no way to get rid of it anymore.

### Your environment

lnd 0.14.1
Linux node 5.10.17 &#35; 1-NixOS SMP Tue Jan 1 00:00:00 UTC 1980 aarch64 GNU/Linux
bitcoind 0.21.0 (bitcoin core)

**Comment by positiveblue** [comment]
You can get information about the tower with `lncli wtclient tower ${TOWER_PUB_KEY} --include_sessions`.

Do not forget to include the `--include_sessions` flag so you get the information about the `NumBackups`, `NumPendingBackups` and the `MaxBackups` (watch tower policy).

I am not aware of any cmd for flushing the last status to the tower (@Roasbeef ?) However, the client will try to send any pending update in the queue to the tower when the server is shutting down. After stopping lnd you may be able to restart it and remove the tower without having to disable the wtclient in the config (check the logs before starting the server again to see if there is any related errors).

**Comment by Roasbeef** [comment]
Closing as a duplicate of: https://github.com/lightningnetwork/lnd/issues/4420
